test/Parse/ConditionalCompilation/decl_parse_errors.swift

```diff
@@ -31,10 +31,10 @@ class C { // expected-note 2 {{in declaration of 'C'}} expected-note {{to match
 
 #if os(iOS)
 	func foo() {}
-} // expected-error{{expected declaration}} expected-error{{expected #else or #endif at end of conditional compilation block}}
```

Comment:
I'm sad to lose these errors. They're for people who have a brain flutter and think `}` will close `#if`. It might be worth the improvement, though.